runtime/runtime-params-estimator/Cargo.toml

```diff
@@ -28,8 +28,10 @@ neard = { path = "../../neard" }
 rocksdb = { git = "https://github.com/nearprotocol/rust-rocksdb", branch="disable-thread" }
 glob = "0.3.0"
 walrus = "0.18.0"
+dynasm = "1.0.0"
```

Comment:
It seems that you use a re-exported version from `dynasmrt`, do you still need it as an explicit dependency?

Reply:
Found a way around it, now just use rust's file read with a raw fd, which the generated assembly was doing with a syscall.